Jenkinsfile

```diff
@@ -108,12 +108,6 @@ node {
             Utils.markStageSkippedForConditional("clangDebugAddrUBSanitizers")
           }
         }
-      }, gccDebug: {
```

Comment:
Der `sh "./gcc-debug/hyriseTest gcc-debug"` wurde jetzt entfernt?

Reply:
Ei dont anderständ juh?!

`gcc-debug/hyriseTest` is already executed before. This was erroneously in there twice.

Reply:
Such improvements. Is matser inspired by arbeist?